### PR TITLE
[UG] Find command step by step example

### PR DESCRIPTION
**Pull request by lingshanng**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -582,7 +580,10 @@ Examples:
 
 **Example of finding by different types of fields**
 
-To find students with the tag `new` or with lessons between `1100-1300`, you may type the command `find cond/any t/new time/1100-1300`. The figure below shows the list after this find command is executed.
+Suppose you want to find students with the tag `new` or with lessons between `1100-1300`.
+1. Type the command `find cond/any t/new time/1100-1300` into the command box. 
+   
+2. Press <kbd>Enter</kbd> and you should see that the list now displays only students matching the given keywords.
 
 ![find](images/find.png)
```
Comment (angkoonhwee) [inline docs/UserGuide.md:588]: fuzzy fuzzy

File: docs/UserGuide.md [hunk 2]
```diff
@@ -582,7 +580,10 @@ Examples:
 
 **Example of finding by different types of fields**
```
Comment (angkoonhwee) [inline docs/UserGuide.md:581]: hmmm for the rest of us we just simply used `Example` should we coordinate?

File: docs/UserGuide.md (hunk 1 above)
Comment (lingshanng) [reply docs/UserGuide.md:596]: is it better?

**Review by Xiaoyunnn — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (angkoonhwee) [reply docs/UserGuide.md:596]: yesss

**Pull request merged**